src/backend/index.js

```diff
@@ -21,4 +23,39 @@ app.use("/api/book", bookRouter);
 
 const port = process.env.PORT || 8000;
 
-// TODO: Connect to PostgreSQL
+// Connect to PostgreSQL
+const client = new Client({
+  host: process.env.PGHOST,
+  user: process.env.DB_USER,
+  database: process.env.PGDATABASE,
+  password: process.env.PGPASSWORD,
+  port: process.env.PGPORT,
+  connectionTimeoutMillis: 5000,
+});
+
+await client.connect()
+  .then(() => {
+    // Successfully connected to PostgreSQL server
+    app.listen(port, () =>
+      console.log(`Server running on port http://localhost:${port}`),
+    );
+    console.log("Connected to PostgreSQL server");
+
+
+    app.get('/', async (req, res) => {
```

Comment:
Add a comment to remind us to delete this

Reply:
Wait I found that I somehow pushed a version without deleting this; I'll fix it later.